### PR TITLE
Updating vera docs to use config flow in related pr.

### PR DESCRIPTION
**Pull request by vangorra**
**Description:**
Changing documentation to prefer config flow instead of configuration.yml.

**Pull request in home-assistant (if applicable):** home-assistant/home-assistant#29880

## Checklist:

- [X] Branch: `next` is for changes and new documentation that will go public with the next Home Assistant release. Fixes, changes and adjustments for the current release should be created against `current`.
- [X] The documentation follows the [standards][standards].

[standards]: https://developers.home-assistant.io/docs/documentation_standards.html

### CODE REVIEW

File: source/_integrations/vera.markdown
Comment (frenck) [inline source/_integrations/vera.markdown]: ~~Let not make big warnings. Those are for the release notes and changelogs that go with the releases.~~

~~Instead, we should write the documentation for a new user getting here. What does he has to click and do to make it work?~~

~~Please write the documentation from a new user perspective.~~

EDIT: 🤦‍♂ Sorry about that. That is incorrect judgment, I need to call it a day. 😉 

However, let's remove this warning.

**Review by frenck — CHANGES_REQUESTED** [review]

**Review by frenck — DISMISSED** [review]
✅ Approved. Can be merged as soon as the parent PR gets merged.

**Comment by vangorra** [comment]
Rebased and linked PR is merged. This should be ready to merge once build finishes.

**Review by frenck — APPROVED** [review]
✅

**Pull request merged**
